scripts/validate-packages.py

```diff
@@ -57,6 +57,15 @@ def _validate_revision(given_package, revision, path):
 
     packaging_version = package_json.get("packagingVersion", "2.0")
 
+    # validate upgrades version
+    if packaging_version == "4.0":
+        min_dcos_version = package_json.get("minDcosReleaseVersion").split('.')
+        major = int(min_dcos_version[0])
+        minor = int(min_dcos_version[1])
+
+        if major < 1 or (major == 1 and minor < 10):
```

Comment:
Let's follow the existing conventions for comparing release versions in this code: https://github.com/larioj/universe/blob/7465bacd4c151afde330bbfffc70832dd7854f46/scripts/validate-packages.py#L109